core/policy/service.go

```diff
@@ -366,3 +371,19 @@ func structToMap(item interface{}) (map[string]interface{}, error) {
 
 	return result, nil
 }
+
+func (s *Service) validateAppealConfig(cfg *domain.PolicyAppealConfig) error {
+	if cfg != nil && cfg.AllowActiveAccessExtensionIn != "" {
+		if err := validateDuration(cfg.AllowActiveAccessExtensionIn); err != nil {
+			return fmt.Errorf("invalid appeal extension policy: %v", err)
```

Comment:
error wrapping with `%w` ?
```suggestion
			return fmt.Errorf("invalid appeal extension policy: %w", err)
```